pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -56,7 +67,7 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 			planDisplay := fmt.Sprintf("Plan %s (%s strategy) [%s]%s", name, plan.Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message))
 			planBranchName := rootBranchName.AddBranch(planDisplay)
 			for _, phase := range lastPlanStatus.Phases {
-				phaseDisplay := fmt.Sprintf("Phase %s [%s]%s", phase.Name, phase.Status, printMessageIfAvailable(phase.Message))
+				phaseDisplay := fmt.Sprintf("Phase %s (%s strategy) [%s]%s", phase.Name, getPhaseStrategy(phase.Name), phase.Status, printMessageIfAvailable(phase.Message))
```

Comment:
Why do we need `getPhaseStrategy` here and not in the [NOT ACTIVE] branch below? There we're directly using phase.Strategy.

Reply:
Because here we're iterating through a struct that doesn't have the strategy property (my understanding is that we store plan status in the instance CRD), whereas below it's a different type that's part of the OperatorVersion and which does have that property.

Reply:
it's because steps don't have strategy

Reply:
@alenkacz But this is a `phase`, not a `step`. 
@yankcrime That makes sense. Basically, we don't have a `phase` here, but a `phaseStatus`, and not a `step` but a `stepStatus`, is that correct? Maybe it'd make sense to change the varNames accordingly?

Reply:
I'm fine with the var names as they are and I think `phaseStatus.status` would look strange (if I'm understanding your point), but I'm happy to defer to someone else if they agree they should be renamed.

Reply:
Right. `phaseStatus.status` would indeed look a bit silly, although it might be clearer that the `phase` in this case is not actually a `phase` but a `phaseStatus` struct. Eh, i guess we can't have it all :)
I'm ok with the names as they are.